pages/clubs.jsx

```diff
@@ -52,6 +59,7 @@ var ClubListItem = React.createClass({
         <p><em>{club.location.split(',')[0]}</em></p>
         <p>{club.description}</p>
         <p><small>Led by <a href={"https://webmaker.org/en-US/search?type=user&q=" + club.owner}>{club.owner}</a></small></p>
+        {status}
```

Comment:
I personal would like a "pending" label shown next to `<h4>{clubName}</h4>`, similar to what we have on the popup on Map.   It makes the labelling more consistent across the page and make it easier to spot "pending" clubs in a long list.